tests/testthat/_snaps/listing_pkpl04.md

```diff
@@ -3,64 +3,64 @@
-      AB12345-CHN-1-id-235          154.052741963676                      NA                             NA                25.0709434529466          NA                      NA             
-       AB12345-CHN-1-id-25          125.321242302376                      NA                             NA                39.5931125402858          NA                      NA             
-       AB12345-CHN-1-id-26          213.205627768882                      NA                             NA                42.5441267818179          NA                      NA             
-      AB12345-CHN-1-id-275          262.577181446438                      NA                             NA                31.3164399296619          NA                      NA             
-      AB12345-CHN-1-id-277          155.38983070326                       NA                             NA                23.1057057955671          NA                      NA             
-      AB12345-CHN-1-id-279          223.565243396403               185.175550222185               1.20731512949823         21.5051531786774   20.6368628167327        1.04207472665083      
-      AB12345-CHN-1-id-287          179.06512202153                       NA                             NA                27.6830100190967          NA                      NA             
-      AB12345-CHN-1-id-312          227.846698887359               188.942155717744               1.20590716254838         29.5814818727077   34.0043190223245        0.869933076833178     
-      AB12345-CHN-1-id-315          226.600898880219                      NA                             NA                37.2950587768686          NA                      NA             
-      AB12345-CHN-1-id-316          199.393501681307                      NA                             NA                32.1545537098299          NA                      NA             
+      ———————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+                     AUC All (h*ug/ml)     AUC All (h*ug/ml)                AUC All (h*ug/ml)                Max Conc (ug/ml)       Max Conc (ug/ml)                 Max Conc (ug/ml)            
+      Subject ID    Xanomeline Low Dose   Xanomeline High Dose   Xanomeline Low Dose/Xanomeline High Dose   Xanomeline Low Dose   Xanomeline High Dose   Xanomeline Low Dose/Xanomeline High Dose
+      ———————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+      01-701-1028           NA              18.1210409513286                        NA                              NA              1.77185469787668                        NA                   
```

Comment:
Are the 4th and 7th columns expected to be all NA here?

Reply:
@edelarua it's actually kinda how the old snapshot looked too, the "issue" with this dataset is that its the same compound for both arms just in different doses, vs the previous is was two different compounds